jira/client.py

```diff
@@ -1123,14 +1139,17 @@ def custom_field_option(self, id: str) -> CustomFieldOption:
     # Dashboards
 
     def dashboards(
-        self, filter=None, startAt=0, maxResults=20
+        self, filter=None, startAt=0, maxResults=20, batch_size: Optional[int] = None
```

Comment:
I added the new argument to the end of parameters of the user-facing methods to keep everything backward compatible. Might be better to adjust it for a breaking release (this method is fine, others are not).